src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(event.target.value);
+  };
```

Comment:
I think the function is extra in this case. Would be simplier to use a callback with setSearchQuery in onChange attribute.

Reply:
why?
But what if you want to give some validation, what you will do?